INZFS.MVC/Views/QuestionDescription/BusinessProposal/_BusinessUploadCostAndPerformanceFigures.cshtml

```diff
@@ -1,3 +1,3 @@
 <p class="govuk-body govuk-!-font-weight-bold">You need to upload evidence to support the claims you've made in <span class="govuk-colour--blueunderline">step 5.7.</span></p>
 
-<p class="govuk-body">You can download a <a href="#" class="govuk-link govuk-!-font-weight-regular">spreadsheet template</a> to show the timeline of cost and performance improvements your technology is likely to have.</p>
\ No newline at end of file
+<p class="govuk-body">You can download a <a href="../media/EEF9 Q5.8. Cost and performance pathway template.xlsx" download class="govuk-link govuk-!-font-weight-regular">spreadsheet template</a> to show the timeline of cost and performance improvements your technology is likely to have.</p>
```

Comment:
I recommend using tilde (root operator).

Example : href="~/media/EEF9 Q5.8. Cost and performance pathway template.xlsx"